css/style.css

```diff
@@ -0,0 +1,202 @@
+  .topnav {
+    background-color: #e9e9e9;
+    display: inline-flex;
+    width: 100%;
+  }
+  
+  #srchbox input {
+    height: 30px;
+    width: 250px;
+  }
+
+
+  #Toolbar {
+    float: right;
+    height: 50px;
+  }
+
+
+
+  #UTube, #srchbox, #sprites {
+
+    display: inline;
+
+  }
```

Comment:
well ids are used for unique purpose. Please read [this](https://www.quackit.com/css/tutorial/css_id.cfm) article for more info